modules/cms-kit/src/Volo.CmsKit.Admin.HttpApi/Volo/CmsKit/Admin/Comments/CommentAdminController.cs

```diff
@@ -47,4 +47,35 @@ public virtual Task DeleteAsync(Guid id)
     {
         return CommentAdminAppService.DeleteAsync(id);
     }
+
+    [HttpPut]
+    [Route("{id}/approval-status")]
+    [Authorize(CmsKitAdminPermissions.Comments.Update)]
```

Comment:
No need to use the `Authorize` attribute in the controllers, because the calling method `UpdateApprovalStatusAsync` has already the related authorization check in the application service implementation. See:

 https://github.com/abpframework/abp/pull/19919/files#diff-ddf2caf499f7695e4b5812196ffef0c08f86e6432eb0912e758f72a0cafc3463R88

So, please remove the authorize attributes in this class.